admin/admin.js

```diff
@@ -0,0 +1,132 @@
+(function () {
```

Comment:
Since we're using vanilla JS, we should be sure to test this cross browser and make sure we're good back to IE10.